tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -5693,8 +5703,8 @@ else if (multimediaDisplayType == 3)
 		} finally {
```

Comment:
The `securityService.pushAdvisor()` is in a different method which I think will result in the advisor being present for the first file, but not any others.